generators/app/templates/package.json

```
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "gulp": "3.9.1",
+    "gulp": "^4.0.0",
```

There is already v4.0.2 

```suggestion
    "gulp": "^4.0.2",
```